GatherPress/gatherpress
# Open Street Maps Implementation

### Description of the Change
This PR addresses the addition of Open Street Maps in Gatherpress. This is a working draft for the OSM team to develop on.
<img width="663" alt="Screenshot 2024-04-06 at 6 24 29 PM" src="https://github.com/GatherPress/gatherpress/assets/28283226/44930cea-5c78-4cf9-ad77-241800b9b1cc">

Closes #561

### How to test the Change
This change can be tested by viewing any event page with a venue containing a location. You should immediately see the OSM version of the map.

### In Progress:
- Geocoding API integration
- Determination of full Google Deprecation (let's save this discussion for Friday after more testing)
- Test updates

### Changelog Entry
> Added - Open Street Maps

### Credits
@stephenerdelyi, @jmarx, @mauteri 

### Checklist:
